pom.xml

```diff
@@ -37,10 +37,11 @@
 		<project.parent.version>${project.version}</project.parent.version>
 		<!-- Dependency versions -->
 		<spring-cloud-dependencies.version>2023.0.2</spring-cloud-dependencies.version>
```

Comment:
https://github.com/GoogleCloudPlatform/spring-cloud-gcp/actions/runs/9651092003/job/26654930547?pr=2990
```
Error: Could not find option 'UnlockExperimentalVMOptions' from 'jar:file:///home/runner/.m2/repository/com/google/cloud/sql/jdbc-socket-factory-core/1.19.0/jdbc-socket-factory-core-1.19.0.jar!/META-INF/native-image/com.google.cloud.sql/cloud-sql-jdbc-socket-factory-parent/native-image.properties'. Use -H:PrintFlags= to list all available options.
Error: Image build request failed with exit status 1
13:25:46:419 [INFO] ------------------------------------------------------------------------
13:25:46:419 [INFO] Reactor Summary for Spring Framework on Google Cloud Code Sample - Postgres 5.4.2-SNAPSHOT:
13:25:46:419 [INFO] 
13:25:46:419 [INFO] Spring Framework on Google Cloud Code Sample - Postgres FAILURE [ 13.090 s]
```

Reply:
https://github.com/GoogleCloudPlatform/cloud-sql-jdbc-socket-factory/releases/tag/v1.19.0

Release notes:
> Update native-image.version to v0.10.2 ([dbffbf4](https://github.com/GoogleCloudPlatform/cloud-sql-jdbc-socket-factory/commit/dbffbf48c38b25f47b85fa50e4b938ebf88f9357))

Reply:
`UnlockExperimentalVMOptions` enabled here:
https://github.com/GoogleCloudPlatform/cloud-sql-jdbc-socket-factory/blob/38b2b1825df3e43ed5775c8cc894aaa037f6a347/core/src/main/resources/META-INF/native-image/com.google.cloud.sql/cloud-sql-jdbc-socket-factory-parent/native-image.properties#L2

Reply:
Native tests skipped in release PR and main branch:
![image](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/assets/40617934/e03b28a6-52a4-421b-a249-91d6d04792ac)
![image](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/assets/40617934/485466d0-bfb8-41ee-b05e-005869720127)

Reply:
Introduced in https://github.com/GoogleCloudPlatform/cloud-sql-jdbc-socket-factory/pull/1961

Reply:
Flag was introduced into GraalVM CE @ 23.1.1: https://github.com/oracle/graal/blob/vm-ce-23.1.1/substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/SubstrateOptions.java#L1015-L1016

Spring Cloud GCP tests with GraalVM 22.3.2: https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/main/.github/workflows/NativeTests.yaml#L84

Reply:
According to https://github.com/graalvm/container/pkgs/container/graalvm-ce, the latest available docker image for graalvm-ce is 22.3.3.

GraalVM community switches to JDK-based versioning.

Reply:
cloud-sql-jdbc-socket-factory is performing scheduled native tests against graalvm-jdk-17.0.11+7.1

https://github.com/GoogleCloudPlatform/cloud-sql-jdbc-socket-factory/actions/runs/9655336140/job/26631058934